plugins/connection/aws_ssm.py

```diff
@@ -48,6 +48,10 @@
     description: The name of the S3 bucket used for file transfers.
     vars:
     - name: ansible_aws_ssm_bucket_name
+  bucket_region:
+    description: The region of the S3 bucket used for file transfers.
+    vars:
+    - name: ansible_aws_ssm_bucket_region
```

Comment:
Could you add please `version_added: 2.0.0`?

Reply:
all set.  Thank you